PART_1_SHARE_INFO.md

```diff
@@ -265,7 +265,7 @@ Each contact info page should:
 - Be named `<your_names>.md`, e.g., `Pat_and_Chris.md`
 - Contain at least:
   - Your preferred names (e.g., "Nic" or "KK")
-  - [Your preferred pronouns](https://www.mypronouns.org/)
+  - [Your pronouns](https://www.mypronouns.org/)
```

Comment:
The one thing I wasn't sure about was whether to retain the link to https://www.mypronouns.org/ in the instructions. I think it's still worth keeping, but I'd love to hear from @kklamberty on this.

Reply:
I think we should keep the link.